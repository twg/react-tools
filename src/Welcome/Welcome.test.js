/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'
import { Welcome } from './'

describe('Welcome', () => {
  it('matches snapshot', () => {
    let component = renderer.create(
      <Welcome />
    )

    expect(component.toJSON()).toMatchSnapshot()
  })
})
