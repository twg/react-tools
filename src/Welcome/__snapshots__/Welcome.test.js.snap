// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome matches snapshot 1`] = `
<div>
  <h1>
    Welcome to the React Component Library
  </h1>
  <p>
    This is a visual library and playground for common components used by TWGers that leverages 
    <a
      href="https://storybook.js.org/"
    >
      Storybook
    </a>
  </p>
  <p>
    A story is a single state of one or more UI components. You can have as many stories as you want.
    <br />
    (Basically a story is like a visual test case.)
  </p>
  <p>
    This component library is most effective when used hand in hand with TRT, which you can find additional documentation for on
     
    <a
      href="https://github.com/twg/react-component-library"
    >
       
      our github page
    </a>
    .
  </p>
  <p>
    Components in this library are typically smaller UI pieces that are very agnostic and should be easy to override / customize.
  </p>
  <p>
    Have a look at the
     
    <a
      href="https://storybook.js.org/basics/writing-stories"
      rel="noopener noreferrer"
      target="_blank"
    >
      Writing Stories
    </a>
     
    section in the storybook documentation for story examples.
  </p>
</div>
`;
